Engine/ScriptAnalyzer.types.ps1xml

```diff
@@ -50,7 +50,13 @@
       <ScriptProperty>
         <Name>Justification</Name>
         <GetScriptBlock>
```

Comment:
I wonder if there should be a boolean property which indicates whether or not this diagnostic record has been suppressed.

Reply:
Hi, James.
SuppressedRecord inherits from DiagnosticRecord.
So one diagnostic record doesn't have the 'Justification' attribute.